packages/Table/src/index.tsx

```diff
@@ -63,35 +65,48 @@ const Table: FunctionComponent<ITableProps> = ({
     onSelect(selectedRows);
   };
 
+  const getSortDirection = (column: string): TSortDirection => {
+    if (sortColumns[column] === hide) {
+      return asc;
+    }
+    return sortColumns[column] === asc ? desc : asc;
+  };
+
   const sort = (column: any) => {
     const resetSortColumns = {};
 
-    Object.keys(sortColumns).forEach(key => (resetSortColumns[key] = asc));
+    Object.keys(sortColumns).forEach(key => (resetSortColumns[key] = hide));
+
+    const sortDirection = getSortDirection(column);
 
-    setRows([..._orderBy(allRows, [column], [sortColumns[column]])]);
+    setRows([..._orderBy(allRows, [column], [sortDirection])]);
 
     setSortColumns({
       ...resetSortColumns,
-      [column]: sortColumns[column] === asc ? desc : asc
+      [column]: sortDirection
     });
   };
 
-  const enableOrderBy = (column: string): JSX.Element => (
-    <Fragment>
-      {orderBy.includes(column) && (
-        <i
+  const enableOrderBy = (column: string): JSX.Element => {
+    return (
```

Comment:
You dont need the return here, you could delete curly braces and add instead parentesis, and then delete return